lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -72,6 +154,26 @@ function cmd.cursePlayers( callingPlayer, targetPlayers, effectName, durationMin
         return
     end
 
+    if smallestAmount == largestAmount and smallestAmount == 1 then
+        amount = false
+    end
```

Comment:
If (due to effect incompatibilities) the highest effect count of any given player was 1, then treat the printout as though `ulx curse target random` was the original command.